client/src/App.js

```diff
@@ -1,5 +1,5 @@
 import { Route, Routes } from "react-router-dom";
-import ModalTestingPage from "./pages/ModalTestingPage";
+//import ModalTestingPage from "./pages/ModalTestingPage";
```

Comment:
Rather than commenting it out, go ahead and delete the code. You can also delete the testing page itself. All the code is stored on GitHub so we can look back at it if we need it later.